posthog/batch_exports/service.py

```diff
@@ -399,6 +399,7 @@ def sync_batch_export(batch_export: BatchExport, created: bool):
                 )
             ),
             id=str(batch_export.id),
+            # TODO: should have batch exports specific task queue
```

Comment:
TODO for @tomasfarias. Is the current task queue just the default setting/do you want to change this? This setting is different for different workers now so the web workers need to be specific about which task queue to add to

Reply:
Batch exports should always run in the same task queue as you defined in the `constants.py` module. Unfortunately, we chose a bad name for it as we were just starting out and not really thinking about more task queues jumping in.

Anyways, fine to set this to `posthog.constants.BATCH_EXPORTS_TASK_QUEUE`.